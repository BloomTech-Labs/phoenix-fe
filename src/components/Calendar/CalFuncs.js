import React from 'react';

export function RenderMonthLabel(props) {
    const currentMonthView = props.selectedMonth;
    return (
      <span className="box month-label">
        {currentMonthView.format("MMMM YYYY")}
      </span>
    );
  }

export function RenderDayLabel(props) {
    const currentSelectedDay = props.selectedDay;
    return (
      <span className="box month-label">
        {currentSelectedDay.format("DD MMMM YYYY")}
      </span>
    );
  }

export function RenderTodayLabel(props) {

    return (
      <span className="box today-label" onClick={props.goToCurrentMonthView}>
        Today
      </span>
    );
  }
  