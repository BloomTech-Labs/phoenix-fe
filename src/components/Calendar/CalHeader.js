import React, { useState } from "react";
import { format, addMonths, subMonths } from "date-fns";

const CalHeader = props => {
  const dateFormat = "MMMM yyyy";

  const nextMonth = () => {
    props.setCurrDate(addMonths(props.currDate, 1));
  };
  const prevMonth = () => {
    props.setCurrDate(subMonths(props.currDate, 1));
  };
  return (
    <div className="calheader row flex-middle">
      <div className="column col-start">
        <div className="icon" onClick={prevMonth}>
          chevron_left
        </div>
      </div>
      <div className="column col-center">
        <span>{format(props.currDate, dateFormat)}</span>
      </div>
      <div className="column col-end">
        <div className="icon" onClick={nextMonth}>
          chevron_right
        </div>
      </div>
    </div>
  );
};

export default CalHeader;