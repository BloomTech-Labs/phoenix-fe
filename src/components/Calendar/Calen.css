body {
  font-size: 16px;
}

.calendar-rectangle {
  width: 100%;
  height: 90%;
  position: relative;
  margin-left: auto;
  margin-right: auto;
  color: #616161;
  font-size: 1em;
  font-family: 'Dosis', sans-serif;
  overflow: hidden;
  box-shadow: 0px 0px 50px #888888;
}
@media (min-width: 576px) {
  .calendar-rectangle {
    width: 70%;
  }
}
@media (min-width: 768px) {
  .calendar-rectangle {
    width: 70%;
    font-size: 1em;
  }
}
@media (min-width: 992px) {
  .calendar-rectangle {
    width: 90%;
    font-size: 1em;
  }
}
@media (min-width: 1200px) {
  .calendar-rectangle {
    width: 90%;
    font-size: 1em;
  }
}
@media (min-width: 1300px) {
  .calendar-rectangle {
    width: 90%;
  }
}
.calendar-rectangle:before {
  content: "";
  display: block;
  padding-top: 120%;
}

.calendar-content {
  position: absolute;
  height: 90%;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
}

.main-calendar {
  flex: 1 0 auto;
  height: 100%;
  flex-wrap: wrap;
}
.main-calendar .calendar-header .title-header {
  width: 100%;
  height: 70%;
  white-space: nowrap;
  font-size: 1.2em;
  background-color: #ffffff;
}
@media (min-width: 992px) {
  .main-calendar .calendar-header .title-header {
    font-size: 1.4em;
  }
}
@media (min-width: 1200px) {
  .main-calendar .calendar-header .title-header {
    font-size: 1.2em;
  }
}
.main-calendar .calendar-header .title-header .header-text {
  flex: 5;
  display: flex;
  height: 100%;
}
.main-calendar .calendar-header .title-header .header-text .today-label {
  flex: 1;
  font-size: 0.8em;
}
.main-calendar .calendar-header .title-header .header-text .today-label:hover {
  cursor: pointer;
  color: #f3214f;
  background-color: #fc9856;
}
.main-calendar .calendar-header .title-header .header-text .month-label {
  flex: 3;
}
.title-header {
  padding-top: 1rem;
}
i {
  color: white;
}
.main-calendar .calendar-header .days-header {
  width: 100%;
  height: 30%;
  background-color: #ffffff;
}
.main-calendar .calendar-header .button-container {
    height: 44px;
  background-color: #f39821c0;
  align-items: center;
  justify-content: center;
}
.main-calendar .calendar-header .button-container .event-button {
  
  display: flex;
  height: 100%;
 
  border: 2px solid #f39821c0;
  border-radius:5px;
  align-items: center;
  justify-content: center;
}
.main-calendar .calendar-header .button-container .event-button:hover {
  background-color: #f3214f;
  color: #f3214f;
}
.main-calendar .days-container {
  width: 100%;
  height: 70%;
  background: #fc9856;
}
.main-calendar .days-container .week {
  height: 15%;
}
.main-calendar .day-events {
  margin: 0 auto;
  padding: 2rem;
  background-color: #f3214f;
  color: white;
  font-weight: bold;
  border-radius: 2px;
}
.main-calendar .day-events .event-container {
  width: 100%;
  text-align: center;
  display: flex;
}
.main-calendar .day-events .event-container:hover {
  cursor: pointer;
}
.main-calendar .day-events .event-container .animated-time {
  width: 30%;
}
.main-calendar .day-events .event-container .animated-title {
  
  
  width: 70%;
}
.main-calendar .day-events .event-container .event-attribute {
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  overflow: hidden;
  box-sizing: border-box;
  padding: 5px;
  
}

.row {
  display: flex;
  justify-content: center;
  align-items: center;
}

.box {
  /* flex-grow: 1; */
  display: flex;
  justify-content: center;
  align-items: center;
  flex: 1;
  height: 100%;
  transition: all 0.4s ease-in-out 0s;
}
.box.arrow:hover {
  background-color: white;
  cursor: pointer;
  color: #fc9856;
  transition: all 0.2s ease-in-out 0s;
}

.day {
  display: flex;
  justify-content: center;
  align-items: center;
  flex: 1;
  height: 100%;
}
.day .day-number {
  width: 80%;
  height: 90%;
  display: flex;
  justify-content: flex-end;
  align-items: flex-start;
  border: 1px solid #f3214f81;
  box-sizing: border-box;
  border-radius: 5%;
}
.day:hover .day-number {
  cursor: default;
  background-color: #f3214f9a;
  color: #ffffff;
  transition: background-color 0.2s ease-in-out 0s;
}
.day.today .day-number {
  border: 1px solid #750019a6;
}
.day.has-events .day-number {
  color: #580215;
  font-weight: bold;
}
.day.has-events .day-number .event {
  max-width: 40%;
  max-height: 80%;
  float: left;
  margin: auto;
  overflow: hidden;
}
.day.different-month {
  opacity: 0.5;
}

/* time animation */
.time-appear {
  transform: translateX(-100%);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.time-appear.time-appear-active {
  transform: translateX(0);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.time-enter {
  transform: translateX(-100%);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.time-enter.time-enter-active {
  transform: translateX(0);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.time-leave {
  transform: translateX(-100%);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.time-leave.time-leave-active {
  transform: translateX(0);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

/* title animation */
.title-appear {
  transform: translateX(100%);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.title-appear.title-appear-active {
  transform: translateX(0);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.title-enter {
  transform: translateX(100%);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.title-enter.title-enter-active {
  transform: translateX(0);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.title-leave {
  transform: translateX(100%);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.title-leave.title-leave-active {
  transform: translateX(0);
  transition: all 0.5s cubic-bezier(0.645, 0.045, 0.355, 1) 0s;
}

.month-arrow {
  padding: 1rem;
}