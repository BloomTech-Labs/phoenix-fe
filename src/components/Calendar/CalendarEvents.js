import React from 'react'
//change props display make better card
const CalendarEvents = (props) => {
    console.log('props', props)
    return (
        <div>
        <h1>Event Name:{props.event.first_name}</h1>   
        <p> Event summary / Description</p> 
        <p>Locaton</p>
        <p>start</p>
        <p>End</p>
        </div>
    )
}

export default CalendarEvents
