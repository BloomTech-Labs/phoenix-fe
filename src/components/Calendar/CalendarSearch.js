import React, {useState, useEffect} from 'react'
import axios from 'axios'
import Event from './Event'

const CalendarSearch = () => {
const [string, setString] = useState('')
const [result, setRes] = useState([])


useEffect(()=> { 
        axios
        .get(`https://phoenix-be-staging.herokuapp.com/api/calendar`)
        .then(res => {
            console.log('card data', res.data)
            const search = res.data.filter(item => {
                return item.summary.toLowerCase().includes(string.toLowerCase())
        })
        setRes(search);})
        .catch(err => console.log('err', err))
    }, [string])

    //in return change to what we want to filter

    const handleChange = e => {
        setString(e.target.value)
    }

    const handleSubmit = (e, {setRes})=> {
        e.preventDefault()
        setRes(e.target.value)
    }

    
    return (
        <div>
            <h3>Search for events</h3>
            <form onSubmit={handleSubmit}>
            <input 
              className="searchInput"
              onChange={handleChange}
              type="text" 
              name="search"
              placeholder="Search"
              autoComplete="off"
              />
             <button type='submit'>Submit</button>
         </form>
         {result.map(data => (
             <Event key={data.id} event={data}/>
         ))}
      
    </div>
    )
}


export default CalendarSearch
