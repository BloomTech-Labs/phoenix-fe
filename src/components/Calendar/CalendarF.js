import React, { useState } from 'react'
import  * as dateFns  from 'date-fns'
import './Calendar.css'
import { makeStyles } from '@material-ui/core/styles';
import Modal from '@material-ui/core/Modal';

//<<<<<<<<<<Single Date Modal Logic>>>>>>>>>>>>
function rand() {
    return Math.round(Math.random() * 20) - 10;
  }
  
  function getModalStyle() {
    const top = 50 + rand();
    const left = 50 + rand();
  
    return {
      top: `${top}%`,
      left: `${left}%`,
      transform: `translate(-${top}%, -${left}%)`,
    };
  }
  
  const useStyles = makeStyles(theme => ({
    paper: {
      position: 'absolute',
      width: 400,
      backgroundColor: theme.palette.background.paper,
      border: '2px solid #000',
      boxShadow: theme.shadows[5],
      padding: theme.spacing(2, 4, 3),
    },
  }));

const CalendarF = () => {
    //<<<<<<<<<<<DATE CALENDR LOGIC>>>>>>>>>>>>>>>>>>
    const [month, SetMonth] =useState(new Date())
    const [week, SetWeek] = useState(new Date())

    const renderHeader = () => {
        const dateFormat = 'MMMM yy'
        return (
            <div className="header row flex-middle">
            <div className="col col-start">
              <div className="icon" onClick={prevMonth}>
                chevron_left
              </div>
            </div>
            <div className="col col-center">
              <span>{dateFns.format(month, dateFormat)}</span>
            </div>
            <div className="col col-end" onClick={nextMonth}>
              <div className="icon">chevron_right</div>
            </div>
          </div>
        )
    }

    const renderDays = () => {
        const dateFormat = "E";
         const days = [];

         let startDate = dateFns.startOfWeek(month);

         for (let i = 0; i < 7; i++) {
           days.push(
             <div className="col col-center" key={i}>
               {dateFns.format(dateFns.addDays(startDate, i), dateFormat)}
             </div>
           );
         }
     
         return <div className="days row">{days}</div>;
    }


   const renderCells= ()=> {
        
        const monthStart = dateFns.startOfMonth(month);
        const monthEnd = dateFns.endOfMonth(monthStart);
        const startWeek = dateFns.startOfWeek(monthStart);
        const endMonthRow = dateFns.endOfWeek(monthEnd);
        const selectedFormat = 'd MM YY a..aaa X'  
        const dateFormat = "d";
        const rows = [];
    
        let days = [];
        let day = startWeek;
        let formattedDate = "";
       
        while (day <= endMonthRow) {
          for (let i = 0; i < 7; i++) {
            
         
           const formattedDate = dateFns.format(day, dateFormat);
            days.push(
              <div
                className={`col cell`}
                key={day}
                onClick={() => onDateClick(days), {handleOpen}}
              >

                {/* <span className="number">{formattedDate}</span> */}
                <span className="bg">{formattedDate}</span>
                <Modal aria-labelledby="simple-modal-title"
        aria-describedby="simple-modal-description"
        open={open}
        onClose={handleClose}
      >
        <div style={modalStyle} className={classes.paper}>
          <p id="simple-modal-description">
            Event to come
          </p>
        </div>
      </Modal>
              </div>
            );
            day = dateFns.addDays(day, 1);
          }
          rows.push(
            <div className="row" key={day}>
              {days}
            </div>
          );
          days = [];
         
          }
      console.log('days', days)
        
        return <div className="body">{rows}</div>;
      }



    const onDateClick = select => {
        console.log('day', select)
     //find the id of a single date poss set to state
   
    };
  
    const nextMonth = () => {
      SetMonth({ month: dateFns.addMonths(month, 1)
      });
    };
  
    const prevMonth = () => {
      SetMonth({
        month: dateFns.subMonths(month, 1)
      });
    };

    //<<<<<<<MODAL LOGIC>>>>>>>>>>>
    const classes = useStyles();
  // getModalStyle is not a pure function, we roll the style only on the first render
  const [modalStyle] = React.useState(getModalStyle);
  const [open, setOpen] = React.useState(false);

  const handleOpen = () => {
    setOpen(true);
  };

  const handleClose = () => {
    setOpen(false);
  };


    return (
        <div>
            {renderHeader()}
            {renderDays()}
            {renderCells()}
        </div>
    )
}

export default CalendarF
