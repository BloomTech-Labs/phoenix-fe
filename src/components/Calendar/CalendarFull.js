import React, { useState } from 'react'
import dateFns from "dateFns";
import './Calendar.css'

const CalendarFull = () => {
    const [date, setDate] = useState(new Date())
    const [month, setMonth] =useState(new Date())

    
    const header = ()=>{
        const dateFormat = 'MMMM YYYY'
        return (
        <div className="header row flex-middle">
        <div className="col col-start">
          <div className="icon" onClick={prevMonth}>
            chevron_left
          </div>
        </div>
        <div className="col col-center">
          <span>
            {dateFns.format(month, dateFormat)}
          </span>
        </div>
        <div className="col col-end" onClick={nextMonth}>
          <div className="icon">chevron_right</div>
        </div>
      </div>
      )
        
    }

    const days= () =>{
        const dateFormat = 'dddd'
        const days = []
        let startDate = dateFns.startOfWeek(month)
        for (let i = 0 ; i < 7; i++){
            days.push(<div className="col col-center" key={i}>
            {dateFns.format(dateFns.addDays(startDate, i), dateFormat)}
          </div>
          )
        }
        return <div className="days row">{days}</div>;
    }
    const onDateClick = day => {
        setDate({selectedDate: day})
    };
    
    const cells =() => {
        //state of date and month
        const monthStart = dateFns.startOfMonth(month);
        const monthEnd = dateFns.endOfMonth(monthStart);
        const startDate = dateFns.startOfWeek(monthStart);
        const endDate = dateFns.endOfWeek(monthEnd);

        const dateFormat = "D";
        const rows = [];
        let days = [];
        let day = startDate;
        let formattedDate = "";
        while (day <= endDate) {
            for (let i = 0; i < 7; i++) {
                formattedDate = dateFns.format(day, dateFormat);
                const cloneDay = day;
                days.push(<div className={`col cell ${!dateFns.isSameMonth(day, monthStart) ? "disabled" : dateFns.isSameDay(day, selectedDate) ? "selected" : "" }`}
                key={day}
                    onClick={() => this.onDateClick(dateFns.parse(cloneDay))} >
                <span className="number">{formattedDate}</span>
                <span className="bg">{formattedDate}</span>
                 </div>
                 );
            day = dateFns.addDays(day, 1);
            }
        rows.push(<div className="row" key={day}>{days}</div>
        );
        days = [];
        }
     return <div className="body">{rows}</div>;
    }
  
   
  
     const nextMonth = () => {
         setMonth(dateFns.addMonths(month, 1))
     };
    const prevMonth = () => {
        setMonth(dateFns.subMonths(month, 1))
    };

    return (
        <div>
            <div>Header</div>
            <div>Days</div>
            <div>Cells</div>
        </div>
    )
}

export default CalendarFull
