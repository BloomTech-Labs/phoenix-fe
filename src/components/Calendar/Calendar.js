import React, { useState } from "react";
import CalHeader from "./CalHeader";
import DaysOfWeek from "./DaysOfWeek";
import Cells from './Cells'
import "./calendar.css";

const Calendar = props => {
  const [currentDate, setCurrentDate] = useState(new Date());
  // const [selectedDate, setSelectedDate] = useState(new Date());

  return (
    <div className="calendar">
      <CalHeader setCurrDate = {setCurrentDate} currDate = {currentDate} />
      <DaysOfWeek currDate = {currentDate}/>
      <Cells currDate = {currentDate} />
    </div>
  );
};

export default Calendar;
