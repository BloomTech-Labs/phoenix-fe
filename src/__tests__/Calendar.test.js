import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer'
import Calendar from '../components/Calendar/Calen'

it("Renders Calender months with dates with specific date events", () => {
    const tree = renderer.create(<Router><Calendar /></Router>).toJSON()

    console.log('tree', tree)

    expect(tree).toMatchSnapshot()
})