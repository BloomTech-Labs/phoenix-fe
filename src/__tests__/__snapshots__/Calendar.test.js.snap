// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Calender months with dates with specific date events 1`] = `
<section
  className="main-calendar"
>
  <header
    className="calendar-header"
  >
    <div
      className="row title-header"
    >
      <div
        className="month-arrow"
        onClick={[Function]}
      >
        <i
          className="box arrow fa fa-angle-left"
        />
      </div>
      <div
        className="box header-text"
      >
        <span
          className="box today-label"
          onClick={[Function]}
        >
          Today
        </span>
        <span
          className="box month-label"
        >
          April 2020
        </span>
      </div>
      <div
        className="month-arrow"
        onClick={[Function]}
      >
        <i
          className="box arrow fa fa-angle-right"
        />
      </div>
    </div>
    <div
      className="row days-header"
    >
      <span
        className="box day-name"
      >
        Mon
      </span>
      <span
        className="box day-name"
      >
        Tue
      </span>
      <span
        className="box day-name"
      >
        Wed
      </span>
      <span
        className="box day-name"
      >
        Thu
      </span>
      <span
        className="box day-name"
      >
        Fri
      </span>
      <span
        className="box day-name"
      >
        Sat
      </span>
      <span
        className="box day-name"
      >
        Sun
      </span>
    </div>
  </header>
  <div
    className="days-container"
  >
    <div
      className="row week"
    >
      <div
        className="day different-month"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          30
        </div>
      </div>
      <div
        className="day different-month"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          31
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          1
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          2
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          3
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          4
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          5
        </div>
      </div>
    </div>
    <div
      className="row week"
    >
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          6
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          7
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          8
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          9
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          10
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          11
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          12
        </div>
      </div>
    </div>
    <div
      className="row week"
    >
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          13
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          14
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          15
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          16
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          17
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          18
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          19
        </div>
      </div>
    </div>
    <div
      className="row week"
    >
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          20
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          21
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          22
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          23
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          24
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          25
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          26
        </div>
      </div>
    </div>
    <div
      className="row week"
    >
      <div
        className="day today selected"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          27
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          28
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          29
        </div>
      </div>
      <div
        className="day"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          30
        </div>
      </div>
      <div
        className="day different-month"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          1
        </div>
      </div>
      <div
        className="day different-month"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          2
        </div>
      </div>
      <div
        className="day different-month"
        onClick={[Function]}
      >
        <div
          className="day-number"
        >
          3
        </div>
      </div>
    </div>
  </div>
</section>
`;
